### DISCUSSION TITLE
Allow assigning machine to group/user from the command line

### DISCUSSION THREAD

**Discussion by b4bit** [rustdesk/rustdesk#6485] [Q&A] [answered]
Is there any way to automatically assign a machine to a group without logging in with any user? We currently need to do it manually, using the Web interface.
A command line like the one to change the fixed password could be a solution: `rustdesk.exe -User jdoe` or `rustdesk.exe -Group test`. Thanks.

**Comment by rustdesk** [comment, marked as answer]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/console/#assign-device-users-and-groups-to-devices
